src/main/java/controller/CarController.java

```diff
@@ -1,5 +1,304 @@
 package controller;
 
+import exception.CarRentalException;
+import lombok.RequiredArgsConstructor;
+import models.cars.Car;
+import models.cars.CarComfort;
+import models.cars.CarType;
+import models.cars.Engine;
+import models.session.Session;
+import services.car.CarComfortService;
+import services.car.CarService;
+import services.car.CarTypeService;
+import services.engine.EngineService;
+import services.session.SessionService;
+import services.user.UserService;
+
+import java.util.List;
+import java.util.Scanner;
+import java.util.UUID;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import static exception.ExceptionHandler.handleException;
+
+@RequiredArgsConstructor
 public class CarController {
+    private final CarService carService;
+    private final UserService userService;
+    private final SessionService sessionService;
+    private final CarComfortService carComfortService;
+    private final CarTypeService carTypeService;
+    private final EngineService engineService;
+    private final Scanner scanner;
+
+
+    public void programInterface() {
+        handleException(() -> {
+            if(sessionService.isUserAuthenticated()){
+                Session session = sessionService.getActive();
+                while (true) {
+                    if (userService.isManager(session.getUserId())) {
```

Comment:
I think you need to separate some actions from regular user and give access only for managers.

Reply:
You mean to make two interfaces - one for users, the second - for managers, and then combine them into one programInterface()?